### PR TITLE
Update com.twocanoes.xcreds.plist

### PR DESCRIPTION
**Pull request by dev-yeet**
added keys for idpHostName and passwordElementID

### CODE REVIEW

File: Manifests/ManagedPreferencesApplications/com.twocanoes.xcreds.plist [hunk 1]
```diff
@@ -199,7 +199,7 @@ A profile can consist of payloads with different version numbers. For example, c
 			<key>pfm_default</key>
 			<false/>
 			<key>pfm_description</key>
-			<string>Save the Personal Recovery Key (PRK) to disk for the MDM Escrow Service to collect.</string>
+			<string>Save the Personal Recovery Key (PRK) to disk for the MDM Escrow Service to collect</string>
```
Comment (apizz) [inline Manifests/ManagedPreferencesApplications/com.twocanoes.xcreds.plist:202]: Can we revert these non-functional changes?  I see no reason to have personal preferences around periods and spaces to be mixed with the core of what this change is, which is adding additional preferences to this manifest.

**Review by apizz — CHANGES_REQUESTED** [review]

File: Manifests/ManagedPreferencesApplications/com.twocanoes.xcreds.plist (hunk 1 above)
Comment (dev-yeet) [reply Manifests/ManagedPreferencesApplications/com.twocanoes.xcreds.plist:202]: i didn't manually make those changes. that's just the plist provided by xcreds on their github

Comment (kevinmcox) [reply Manifests/ManagedPreferencesApplications/com.twocanoes.xcreds.plist:202]: @dev-yeet can you roll them back though?

Comment (apizz) [reply Manifests/ManagedPreferencesApplications/com.twocanoes.xcreds.plist:202]: ^ yeah we don't want functional changes interspersed with non-functional.  If there are discrepancies w/ punctuation and spacing, let's leave that out for a separate PR.

Comment (dev-yeet) [reply Manifests/ManagedPreferencesApplications/com.twocanoes.xcreds.plist]: sorry I missed this, does this work?

File: Manifests/ManagedPreferencesApplications/com.twocanoes.xcreds.plist [hunk 2]
```diff
@@ -466,6 +466,30 @@ A profile can consist of payloads with different version numbers. For example, c
 			<string>Verify Cloud Password</string>
 			<key>pfm_type</key>
 			<string>boolean</string>
+		</dict>
+				<dict>
```
Comment (apizz) [inline Manifests/ManagedPreferencesApplications/com.twocanoes.xcreds.plist:470]: fix tab spacing

File: Manifests/ManagedPreferencesApplications/com.twocanoes.xcreds.plist [hunk 3]
```diff
@@ -466,6 +466,30 @@ A profile can consist of payloads with different version numbers. For example, c
...
+			<key>pfm_description</key>
+			<string>hostname of the page that has the password field.</string>
+			<key>pfm_documentation_url</key>
+			<string>https://github.com/twocanoes/xcreds/wiki/AdminGuide#idphostname</string>
+			<key>pfm_name</key>
+			<string>idpHostName</string>
+			<key>pfm_title</key>
+			<string>idpHostName</string>
```
Comment (apizz) [inline Manifests/ManagedPreferencesApplications/com.twocanoes.xcreds.plist:478]: The point of the `pfm_title` is to be a more user friendly version of what this actually is / does.  So you can either add this user friendly title or remove this `pfm_title` key.

File: Manifests/ManagedPreferencesApplications/com.twocanoes.xcreds.plist [hunk 4]
```diff
@@ -466,6 +466,30 @@ A profile can consist of payloads with different version numbers. For example, c
...
+		</dict>
+		<dict>
+			<key>pfm_description</key>
+			<string>password element id of the html element that has the password.</string>
+			<key>pfm_documentation_url</key>
+			<string>https://github.com/twocanoes/xcreds/wiki/AdminGuide#passwordelementid</string>
+			<key>pfm_name</key>
+			<string>passwordElementID</string>
+			<key>pfm_title</key>
+			<string>passwordElementID</string>
```
Comment (apizz) [inline Manifests/ManagedPreferencesApplications/com.twocanoes.xcreds.plist:490]: Same as previous comment

**Review by apizz — CHANGES_REQUESTED** [review]

File: Manifests/ManagedPreferencesApplications/com.twocanoes.xcreds.plist (hunk 2 above)
Comment (kevinmcox) [reply Manifests/ManagedPreferencesApplications/com.twocanoes.xcreds.plist]: I committed a change to fix this.

File: Manifests/ManagedPreferencesApplications/com.twocanoes.xcreds.plist (hunk 3 above)
Comment (kevinmcox) [reply Manifests/ManagedPreferencesApplications/com.twocanoes.xcreds.plist]: I committed a change to fix this.

File: Manifests/ManagedPreferencesApplications/com.twocanoes.xcreds.plist (hunk 4 above)
Comment (kevinmcox) [reply Manifests/ManagedPreferencesApplications/com.twocanoes.xcreds.plist]: I committed a change to fix this.

File: Manifests/ManagedPreferencesApplications/com.twocanoes.xcreds.plist (hunk 1 above)
Comment (kevinmcox) [reply Manifests/ManagedPreferencesApplications/com.twocanoes.xcreds.plist]: This fix looks good, thanks.

**Comment by apizz** [comment]
We have to get this branch rebased from latest, as there are conflicts from a previous merge

**Pull request merged**